src/store/flow/flow.ts

```diff
@@ -77,9 +77,6 @@ export const mutations: MutationTree<IFlowsState> = {
       const exitsTowardUs = exits.filter((e) => e.destinationBlock === blockId)
       forEach(exitsTowardUs, (e) => e.destinationBlock = undefined)
     })
-
-    // @ts-ignore
-    this.state.builder.activeBlockId = null
```

Comment:
How come this is no longer necessary?

Reply:
Um, IFlowState doesn't have `builder` prop, therefore this is always a TS error. So I moved this out of this mutation https://github.com/FLOIP/flow-builder/pull/55/files#diff-3db5c56652da6d4f7b33c23e432a38b9b73d0b30d47df648929798e4c82d498bR334.

But when thinking longer, I found a solution:
- instead of having a mutation, which doesn't give us access to `commit` or `dispatch`
- I changed it into action. Then I changed this line into `commit('builder/activateBlock', { blockId: null }, { root: true })`

Thanks for noticing

Reply:
put it back https://github.com/FLOIP/flow-builder/pull/55/files#diff-eea673c657b7edcdbf00ab176fcc8ec0a84d21e8a8dfa9ff8a014c7510a144a0R134

Reply:
Okay, cool. I think that should work